programmers.txt

```diff
@@ -18,3 +18,11 @@ cmsisdap.protocol=cmsis-dap
 cmsisdap.program.protocol=cmsis-dap
 cmsisdap.program.tool=openocd
 cmsisdap.program.setup_command=;
+
+bmp.name=Black Magic Probe (GDB)
```

Comment:
Can we change `bmp` to `blackmagicprobe` in this file?

Reply:
Fixed